src/cfn/index.ts

```diff
@@ -761,15 +765,18 @@ async function listStacks(showTags = false, tagsFilter?: [string, string][]) {
     } else if (lifecyle === 'short') {
       lifecyleIcon = '♺ ';
     }
+    const baseStackName = (stack.StackName.startsWith('StackSet-') && stack.Description)
+      ? `${cli.blackBright(stack.StackName)} ${stack.Description}`
```

Comment:
What do you think about looking for a tag (something like `StackSetName`) and using that if present, otherwise fallback to the description?